### ISSUE TITLE
Add meta fields to account schema

### ISSUE THREAD

**Issue by advishnuprasad** [proteanhq/authentic#9]
Meta fields created_by and updated_by are missing in account schema

**Comment by abhishek-ram** [comment]
Yes @advishnuprasad  the idea is to add the bare minimum properties and allow users to extend if needed, right @subhashb ?

**Comment by subhashb** [comment]
That would be true for the Protean core, yes.

But I see the following aspects in Authentic that are different:
- Authentic is an implementation of Protean, and will probably not be inherited. Users will use it for the out-of-the-box functionality, so will probably not tamper with the models.
- Authentic is all about security, and the four meta fields are required to ensure backtracking and audits.
- If Authentic were made available as a "Service" on the cloud, these meta fields would be necessary.

Let us include the four meta fields as part of Authentic. As usual, the correct way would be to create a custom Base Entity, inheriting from Entity, and add the audit fields to it, before inheriting all other entities from the customized Base Entity.
